.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+- package-ecosystem: bundler
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "00:00"
+    timezone: America/New_York
+  open-pull-requests-limit: 99
+  labels:
+  - dependencies
+  - Needs QA
+  allow:
+  - dependency-type: direct
```

Comment:
Super nitpicky but our practice is to have a newline at the end of every file. (There's usually an editor setting you can set to do this automatically.) Would you mind updating this file and the `main.yml` file?